### PR TITLE
Adds deodorant

### PR DESCRIPTION
**Pull request by 1fbff5f83b23d39d38b1dfcb4cac8d9b**
Adds deodorant so you don't have to shower. Thanks to our beloved @Partheo for sprites!

:cl:  McDonald072
rscadd: Nanotrasen now ships deodorant to its stations following the events on Space Station 7.
/:cl:

### CODE REVIEW

**Comment by Alexandria404** [comment]
VERY NICE.

**Comment by alexkar598** [comment]
Is this mapped in/is there a way to get it yet?

**Comment by 1fbff5f83b23d39d38b1dfcb4cac8d9b** [comment]
not yet

**Comment by 1fbff5f83b23d39d38b1dfcb4cac8d9b** [comment]
There are now 2 cans in dorms and you can order 5 for 1000 at cargo

**Comment by MegaEmpirical** [comment]
Could we add a few to the clerk vendor (seems like a natural place to store some and sell) and maybe some in medical?

**Comment by monster860** [comment]
You smell.

**Comment by ThatLing** [comment]
what is space station 7

**Comment by 1fbff5f83b23d39d38b1dfcb4cac8d9b** [comment]
the correct question would be "what was space station 7"

File: yogstation/code/game/objects/items/deodorant.dm [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+			playsound(user, 'yogstation/sound/effects/burning.wav', 50)
+			user.visible_message("<span class='danger'>[user] sprays [target] with fire!</span>", "<span class='danger'><b>BURN BABY BURN!!</b></span>")
+			var/mob/living/simple_animal/hostile/poison/giant_spider/GCS = target
+			GCS.adjustHealth(30)
+			uses--
+		else if(iscarbon(target))
+			playsound(user, 'yogstation/sound/effects/burning.wav', 50)
+			var/mob/living/carbon/C = target
+			user.visible_message("<span class='danger'>[user] sprays [C] with fire!</span>", "<span class='danger'><b>BURN BABY BURN!!</b></span>")
+			to_chat(target, "<span class='userdanger'>[user] sprays you with fire!</span>")
```
Comment (Amelia0010) [inline yogstation/code/game/objects/items/deodorant.dm]: I'm about 99.999999999% certain the target will get both this and the above message.

File: yogstation/code/game/objects/items/deodorant.dm [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+			sleep(30)
+			return OXYLOSS
+		if(2)
+			user.visible_message("<span class='danger'>[user] is huffing [src], it looks like [user.p_theyre()] trying to commit suicide!")
+			return OXYLOSS
+		if(3)
+			user.visible_message("<span class='danger'>[user] is rapidly shaking [src], it looks like [user.p_theyre()] trying to commit suicide!")
+			sleep(15)
+			playsound(user, 'sound/effects/explosion3.ogg', 50)
+			qdel(src)
```
Comment (Amelia0010) [inline yogstation/code/game/objects/items/deodorant.dm:72]: make it do a 0,0,1 explosion as well

**Review by Amelia0010 — DISMISSED** [review]

**Pull request merged**